components/Navigation.tsx

```diff
@@ -473,6 +473,27 @@ const navigation: navigationItem[] = [
       },
     ],
   },
+  {
+    title: 'Avo Public API',
+    iconName: 'rocket',
+    subroutes: [
+      {
+        type: 'route',
+        title: 'Overview',
+        path: '/avoapi/overview',
+      },
+      {
+        type: 'route',
+        title: 'Authentication',
+        path: '/avoapi/authentication',
+      },
+      {
+        type: 'route',
+        title: 'Export Tracking Plan',
+        path: '/avoapi/export-tracking-plan',
+      },
+    ],
+  },
```

Comment:
I'm not sure we want to dedicate a whole section in the sidebar to the public API yet, I'd like to suggest we keep this under the "Reference" tab for now. Any objections to that?

Reply:
Updated in cf0f9079cfa32e8fd33afd4b74e36ce6e38306bd